### DISCUSSION TITLE
Telegram Web and :matches-path(#)

### DISCUSSION THREAD

**Discussion by Vanerio** [uBlockOrigin/uBlock-issues#2353] [General] [answered]
Hi!
I am trying to apply some filters into telegram groups.
Groups have a ` #-` in their URL, but I can´t make uBlock detect this with :matches-path()

So this doesn´t work:
`web.telegram.org##:matches-path(/^/z/#-/)`

uBlock detects # as the end of the URL, because `##:matches-path(/^/z/$/)` matches correctly. 
Is this behaviour normal os should uBlock selection work? Does anyone know how could I accomplish it?

Thank you very much!

**Comment by gorhill** [comment, marked as answer]
`:matches-path` tries to match only the path part of the URL, and `#...`  is not part of the path part, it's part of the anchor part.

**Comment by Vanerio** [comment]
That's what I was afraid of. Thanks gorhill.

So if anyone is pretending the same, here is how I managed to do it:
Start the filter with: `###MiddleColumn:has(.group-status) `
